chapters-find.md

```diff
@@ -15,50 +15,50 @@ The best way to find what chapter you belong to is learning from your troop. You
   </div>
 </form>
 
-<span id="alert-response"></span>
+<div class="alert alert-info fade" role="alert" id="alert-response" hidden>
+  The zipcode you entered aligns with the chapters below. Either use the map to see which chapter you are in or contact one of the leaders from your unit or chapter.
+  <ul id="findchapter-chapters"></ul>
+</div>
 
 <iframe src="https://www.google.com/maps/d/embed?mid=1cEJd5Fv4sfaouJ9mOJ66Hwj2l80&hl=en" width="100%" height="400" frameborder="0" class="mt-3"></iframe>
 
 <script type="text/javascript">
 
 chapters = {{ site.data.chapters | jsonify }};
 
-const input = document.querySelector('input');
-input.addEventListener('input', updateValue);
+const input = document.getElementById("findchapterform-zip");
+const multiple_div = document.getElementById("alert-response");
+const chapters_ul = document.getElementById("findchapter-chapters");
+input.addEventListener('input', updateZip);
 
 var chapters_matched =[];
-var multiple_text = "The zipcode you entered aligns with the chapters below. Either use the map to see which chapter you are in or contact one of the leaders from your unit or chapter.";
+var multi_chapter_output = "";
 
-function updateValue(e) {
-  if(e.target.value.toString().length > 4)
+function updateZip(zip_input) {
```

Comment:
There's a human factors issue here, in that I get zero feedback if I enter an invalid zip.  That said, I'd add this to an issue list and re-visit ASAP - not keep it from going live.